src/index.js

```diff
@@ -9,6 +9,7 @@ import userIsLoggedIn from './utils/userIsLoggedIn';
 import { logOutUser, setLoggedInUser } from './actions/auth';
 
 const store = configureStore();
+console.log('apiUrl', process.env.REACT_APP_API);
```

Comment:
You might want to remove this console.log statement

Reply:
I put this console.log statement so that when the app is running we can know what api it is making ajax requests to.

Reply:
Alright, That makes sense.